### PR TITLE
Various case and text fixes.

### PR DESCRIPTION
**Pull request by alimony**
This fixes some consistency issues with text cases on labels (https://github.com/bluesky-social/social-app/issues/709) and includes a few other tiny text polish fixes. I will make line comments in this PR to clarify a few things.

### CODE REVIEW

File: src/view/com/modals/AddAppPasswords.tsx
Comment (alimony) [inline src/view/com/modals/AddAppPasswords.tsx:1]: This seems like a jumble of two different versions, I used the one used elsewhere in the app.

File: src/view/com/modals/DeleteAccount.tsx [hunk 1]
```diff
@@ -67,13 +67,13 @@ export function Component({}: {}) {
...
         <Text type="title-xl" style={[styles.title, pal.text]}>
-          Delete account
+          Delete Account
         </Text>
         {!isEmailSent ? (
           <>
             <Text type="lg" style={[styles.description, pal.text]}>
               For security reasons, we'll need to send a confirmation code to
-              your email.
+              your email address.
```
Comment (alimony) [inline src/view/com/modals/DeleteAccount.tsx:76]: Wording seems clearer to me.

File: src/view/screens/SearchMobile.tsx [hunk 2]
```diff
@@ -164,7 +164,7 @@ export const SearchScreen = withAuthRequired(
               ) : isInputFocused ? (
                 <View>
                   <Text style={[pal.textLight, styles.searchPrompt]}>
-                    Search for users on the network
+                    Search for users and posts on the network
```
Comment (alimony) [inline src/view/screens/SearchMobile.tsx:167]: To clarify this searches for both users and posts.

File: src/view/screens/Settings.tsx [hunk 3]
```diff
@@ -354,12 +354,12 @@ export const SettingsScreen = withAuthRequired(
               />
             </View>
             <Text type="lg" style={dangerText}>
-              Delete my account
+              Delete my account…
```
Comment (alimony) [inline src/view/screens/Settings.tsx:357]: To follow the convention of system menu items etc. where ellipsis means there will be further prompts before any action is actually taken.

File: src/view/com/auth/SplashScreen.web.tsx [hunk 4]
```diff
@@ -47,7 +47,7 @@ export const SplashScreen = ({
               // TODO: web accessibility
               accessibilityRole="button">
               <Text style={[s.white, styles.btnLabel]}>
-                Create a new account
+                Create a New Account
```
Comment (ansh) [inline src/view/com/auth/SplashScreen.web.tsx:50]: Can we keep this lowercase?

File: src/view/com/auth/SplashScreen.tsx [hunk 5]
```diff
@@ -33,7 +33,7 @@ export const SplashScreen = ({
               accessibilityLabel="Create new account"
               accessibilityHint="Opens flow to create a new Bluesky account">
               <Text style={[s.white, styles.btnLabel]}>
-                Create a new account
+                Create a New Account
```
Comment (ansh) [inline src/view/com/auth/SplashScreen.tsx:36]: Can we keep this lowercase?

File: __e2e__/tests/home-screen.test.ts [hunk 6]
```diff
@@ -72,7 +72,7 @@ describe('Home screen', () => {
   })
 
   it('Can tap between feeds', async () => {
-    await element(by.id("homeScreenFeedTabs-What's hot")).tap()
+    await element(by.id("homeScreenFeedTabs-What's Hot")).tap()
```
Comment (ansh) [inline __e2e__/tests/home-screen.test.ts:75]: Let's revert this for now

File: src/view/com/modals/AddAppPasswords.tsx [hunk 7]
```diff
@@ -184,9 +184,9 @@ export function Component({}: {}) {
         </Text>
       ) : (
         <Text type="xs" style={[pal.textLight, s.mb10, s.mt2]}>
-          Only contain letters, numbers, spaces, dashes, and underscores
-          allowed. Must be at least 4 characters long, but no more than 32
-          characters long.
+          Can only contain letters, numbers, spaces, dashes, and underscores.
```
Comment (ansh) [inline src/view/com/modals/AddAppPasswords.tsx:187]: Good catch! Thanks :)

File: src/view/shell/desktop/RightNav.tsx [hunk 8]
```diff
@@ -46,7 +46,7 @@ export const DesktopRightNav = observer(function DesktopRightNav() {
             type="md"
             style={pal.link}
             href="/support/privacy"
-            text="Privacy Policy"
+            text="Privacy policy"
```
Comment (ansh) [inline src/view/shell/desktop/RightNav.tsx:49]: Let's revert this for now.

**Review by ansh — CHANGES_REQUESTED** [review]
Mostly looks good. Have a few changes.

**Review by ansh — CHANGES_REQUESTED** [review]
Mostly looks good. Have a few changes.

**Comment by ansh** [comment]
@alimony Fixed lint errors on main. Feel free to merge main into this branch to re-runt he tests.

**Comment by alimony** [comment]
@ansh Done and passing!

**Comment by alimony** [comment]
@ansh This is still waiting for approval after making your requested changes.

**Review by ansh — APPROVED** [review]
LGTM. Thanks for the PR!

**Pull request merged**